### PR TITLE
Google Earth now provide an installer

### PR DESCRIPTION
**Pull request by wmorin**
Here is a new attempt after this pull request where I forgot to provide uninstall
https://github.com/caskroom/homebrew-cask/pull/10850

### CODE REVIEW

File: Casks/google-earth.rb [hunk 1]
```diff
@@ -3,14 +3,24 @@
...
+                         'com.google.pkg.Keystone',
+                        ]
+  zap :delete    => [
+                     '~/Library/Application Support/Google Earth',
+                     '~/Library/Caches/com.Google.GoogleEarthPlus',
+                     '~/Library/Caches/com.google.Keystone.Agent',
+                     '~/Library/Caches/Google Earth',
+                     '~/Library/Preferences/com.Google.GoogleEarthPlus.plist',
+                    ],
+      :launchctl => 'com.google.keystone.agent.plist',
```
Comment (vitorgalvao) [inline Casks/google-earth.rb]: This one should be in `uninstall`.

File: Casks/google-earth.rb [hunk 2]
```diff
@@ -3,14 +3,24 @@
...
 
-  zap :delete => [
-                  '~/Library/Application Support/Google Earth',
-                  '~/Library/Caches/Google Earth',
-                  '~/Library/Caches/com.Google.GoogleEarthPlus',
-                 ]
+  uninstall :pkgutil => [
+                         'com.Google.GoogleEarthPlus',
+                         'com.Google.GoogleEarthPlugin.plugin',
+                         'com.google.pkg.Keystone',
```
Comment (vitorgalvao) [inline Casks/google-earth.rb]: This one should be removed, since it’s shared among other Google apps, and removing it could break updating for others.

**Comment by wmorin** [comment]
Thank you Vítor!

**Comment by fanquake** [comment]
This needs a rebase

**Pull request merged**